package.json

```diff
@@ -6,14 +6,29 @@
     "dev": "nuxt",
     "build": "nuxt build",
     "start": "nuxt start",
-    "generate": "nuxt generate"
+    "generate": "nuxt generate",
+    "pretty": "prettier --write \"./**/*.{js,jsx,json,md}\"",
+    "lint": "npm run pretty"
```

Comment:
`lint` is usually not a good NPM script to use. For example, `yarn lint` is a special command, specific to Yarn workflow.

Use something like `lint:format` as a script name.

Also, why `npm run`? Use Yarn v1.

In fact I'm not sure why you'd even need this script. `pretty` script is there in the line above. Most of the times, Prettier invocation would be automated through your editor / IDE, or git hook. If you do have to manually invoke it, use `yarn pretty`.

Reply:
Yes, it can be automated but here husky need the command to run before making a commit so `pretty` is used or if someones IDE is not setup then he can simply use the script to prettify the code.